ex12.py

```diff
@@ -0,0 +1,10 @@
+age = raw_input("How old are you? ")
```

Comment:
The raw_input is basically used to give the input run-time (i.e. while your script is running).